### ISSUE TITLE
Setup for XIVLauncher not working

### ISSUE THREAD

**Issue by Flareling** [goatcorp/FFXIVQuickLauncher#508]
When I run the setup program, all I get is this error message:
![image](https://user-images.githubusercontent.com/88813857/129135475-c7cf9693-fcfa-481a-b727-c5505d4e7efa.png)

It does however create a XIVlauncher folder on my pc containing various files and programs , with a program called updater, although when I run this nothing happens. The file does not seem to contain the actual launcher for XIVLauncher however. 

Additionally, I run the setup as an admin and have set an exception for it in my antivirus (avast).

The first time I installed this program it worked first time, However after deleting it (using the windows uninstaller thing) and trying to reinstall it, it no longer works.

Here is the setup log if it helps:
[SquirrelSetup.log](https://github.com/goatcorp/FFXIVQuickLauncher/files/6972718/SquirrelSetup.log)

Thanks for any help

**Comment by lmcintyre** [comment]
The issue seems to be due to using admin permissions to install it originally. You will have to uninstall it, and delete the XIVLauncher folder from AppData/Roaming and AppData/Local. Nothing except the patch installer requires admin rights on your system (and that's depending on game install location). Running anything XIVLauncher as admin is asking for a bad time.

**Comment by reiichi001** [comment]
Hello! Is this happening, @Flareling?

**Comment by stogie73** [comment]
Hi, I'm having the same issues. It had been working then Bitdefender quarantined it this morning (even though I had all the necessary exceptions). 

I removed it from quarantine but it wouldn't load, so I uninstalled it, deleted the folders and tried to reinstall.
I did try installing it as admin and as well as normally, with and without the folders.

It also doesn't create a folder in AppData/Roaming
[SquirrelSetup.log](https://github.com/goatcorp/FFXIVQuickLauncher/files/7316020/SquirrelSetup.log)
.
